webapp-django/crashstats/settings/test.py

```diff
@@ -1,12 +1,19 @@
-## This is automatically imported by test-utils to make sure tests are run in
-## a consistent way across different platforms and different developers.
+"""
+When you run tests with `./manage.py test` these settings are ALWAYS
+imported last and overrides any other base or environment variable
+settings.
+
+The purpose of this is to guarantee that certain settings are always
+set for test suite runs no matter what you do on the local system.
+
+Ultimately, it helps make sure you never actually use real URLs. For
+example, if a test incorrectly doesn't mock `requests.get()` for
+example, it shouldn't actually try to reach out to a real valid URL.
+"""
 
 CACHE_MIDDLEWARE = True
 CACHE_MIDDLEWARE_FILES = False
 
-import os
-os.environ['FORCE_DB'] = 'true'
```

Comment:
Can you explain this change? It seems like it should never have been this way, are you just fixing it while you're at it?